data/formats-data.js

```diff
@@ -1849,7 +1849,7 @@ exports.BattleFormatsData = {
 		tier: "LC",
 	},
 	noctowl: {
-		randomBattleMoves: ["roost", "whirlwind", "airslash", "nightshade", "toxic", "defog"],
+		randomBattleMoves: ["roost", "whirlwind", "airslash", "nightshade", "toxic", "defog", "moonblast", "hurricane", "hypervoice", "heatwave"],
```

Comment:
Uhm... I suppose Hurricane and Heat Wave are alright.

Reply:
An offensive tinted lens set could work